cargo/Maven2+Plugin+Getting+Started.html

```diff
@@ -291,10 +291,10 @@
            <div class="codeContent panelContent pdl"> 
             <pre class="syntaxhighlighter-pre" data-syntaxhighlighter-params="brush: xml; gutter: false; theme: Confluence" data-theme="Confluence">mvn clean verify org.codehaus.cargo:cargo-maven2-plugin:run
     -Dcargo.maven.containerId=tomcat7x
-    -Dcargo.maven.containerUrl=http://archive.apache.org/dist/tomcat/tomcat-7/v7.0.16/bin/apache-tomcat-7.0.16.zip
+    -Dcargo.maven.containerUrl=http://repo2.maven.org/maven2/org/apache/tomcat/tomcat/7.0.61/tomcat-7.0.61.zip

```

Comment:
Why not `http://central.maven.org/maven2/org/apache/tomcat/tomcat/${tomcat.version}/tomcat-${tomcat.version}.zip` instead?